app/_layouts/extension.html

```diff
@@ -210,19 +287,53 @@
 {% endcapture %}
 
 {% capture plugin_config_for_route %}
-### Enabling the plugin on a Route
+### Enabling the plugin on a route
 
-{% if page.params.yaml_examples == false and page.params.k8s_examples == false %}
+{% if page.params.yaml_examples == false and page.params.k8s_examples == false and page.params.manager_examples == false and page.params.konnect_examples == false %}
 
 {{ plugin_config_for_route_admin_api }}
 
+`<route>` is the `id` or `name` of the route that this plugin configuration
+  will target.
+
 {% else %}
 {% navtabs %}
 {% navtab Admin API %}
 
 {{ plugin_config_for_route_admin_api }}
 
+`<route>` is the `id` or `name` of the route that this plugin configuration
+  will target.
+
+{% endnavtab %}
+{% unless page.params.manager_examples == false %}
+{% navtab Kong Manager %}
+Configure this plugin on a route:
+
+1. In Kong Manager, select the workspace.
+2. From the Dashboard, select **Routes** in the left navigation.
+3. Click **View** for the route row.
+4. Scroll down to plugins and click **Add Plugin**.
+5. Find and select the **{{ page.name }}** plugin.
+
+    <div class="alert alert-ee blue">
+    <strong>Note:</strong> If the plugin is greyed out, then it is not available
+    for your product tier. See <a href="">Kong Gateway tiers</a>.
+    </div>
+
+6. If the option is available, set the plugin scope to **Scoped**.
```

Comment:
```suggestion
6. If the option is available, set the plugin to **Scoped**.
```
Maybe shorten this to avoid repetition? It's not explicitly labelled as a plugin scope section anyway.

Reply:
k, will fix